modules/rpc/yarpc_options.go

```diff
@@ -71,3 +73,20 @@ func onewayInboundMiddlewareFromCreateInfo(mci service.ModuleCreateInfo) []middl
 	// Intentionally panic if programmer adds non-middleware slice to the data
 	return items.([]middleware.OnewayInbound)
 }
+
+func withGraph(graph dig.Graph) modules.Option {
```

Comment:
This is an a private option to a constructor for testing.

What you are proposing @anup is to have a graph to be a global dump of all objects, so we essentially left with the same globals as before(actually worse: we added an extra dependency). This is worse for testing, hard to have multiple objects of the same type, etc